wagtail/admin/templates/wagtailadmin/shared/panel.html

```diff
@@ -0,0 +1,62 @@
+{% load wagtailadmin_tags i18n %}
+{% comment %}
+    Variables this template accepts:
+
+    id_prefix - A prefix for all id attributes.
+    classes - List of CSS classes to use for the panel.
+    class_name - String of CSS classes to use for the panel.
```

Comment:
can we avoid adding even more different approaches for `classname` passing into components?

see https://github.com/wagtail/wagtail/issues/8885 & https://github.com/wagtail/wagtail/pull/8886

Reply:
not sure if we even want to ever accept a list of strings (instead of just a string, space separated) - maybe this is something we need to support  for some reason but I do not think we do that anywhere else.

Reply:
This is done for backwards compatibility only, to support the same methods as were previously available on panel implementations. I’d be happy to refactor this but I’d prefer to avoid this for now (unsure of the exact compatibility implications) if you’re ok with that.

Reply:
As discussed - if possible we should try to pass these in as the `classname` through context at a minimum? But understand we do not want to break existing approaches.

Reply:
Done! I think we should be good for now, with further refactoring on the Python side to consider later.